makefiles/dependency.mk

```diff
@@ -56,20 +56,19 @@ else
 CUE=$(shell which cue)
 endif
 
-KUSTOMIZE_VERSION ?= 4.5.4	
-
+KUSTOMIZE_VERSION ?= 4.5.4
+KUSTOMIZE = $(shell pwd)/bin/kustomize
 .PHONY: kustomize
 kustomize:
-ifeq (, $(shell kustomize version | grep $(KUSTOMIZE_VERSION)))
+ifeq (, $(shell $(KUSTOMIZE) version | grep $(KUSTOMIZE_VERSION)))
 	@{ \
 	set -eo pipefail ;\
-	echo 'installing kustomize-v$(KUSTOMIZE_VERSION) into $(GOBIN)' ;\
-	curl -sS https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh | bash -s $(KUSTOMIZE_VERSION) $(GOBIN);\
+    echo "installing kustomize-v$(KUSTOMIZE_VERSION) into $(shell pwd)/bin" ;\
+    mkdir -p $(shell pwd)/bin ;\
+    rm -f $(KUSTOMIZE) ;\
```

Comment:
Remove the old version after downloading successfully.

Reply:
I think it's okay to remove the old version  first, because the old version  is useless.

Reply:
Yes, users can re-run this command for a new download.